oioioi/contests/models.py

```diff
@@ -101,7 +101,7 @@ class Contest(models.Model):
     )
 
     # Part of szkopul backporting.
-    # This is a hack for situation where contest controller is empty, 
```

Comment:
I know, this trailing whitespace is bad here, but please, consider withdrawing this change to make diff's smaller.